components/button/button.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="button.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+    />
+    <title>avatar</title>
+  </head>
+  <body>
+    <div class="centered-row p-v-3">
+      <button class="btn btn-primary m-h-3 t-c-1">Default</button>
+      <button class="btn btn-accent m-h-3 t-c-1">Accent</button>
+      <button class="btn btn-danger m-h-3 t-c-1">Danger</button>
+      <button class="btn outline-danger m-h-3 t-c-2">Danger</button>
+      <button class="btn btn-pill f-4 btn-accent m-h-3 t-c-1">Pill</button>
+      <button class="btn btn-custom m-h-3 t-c-1">show</button>
+      <button class="btn btn-danger floating m-8 t-c-1">Floating Button</button>
+      <button class="btn btn-link m-h-3 t-c-3">Link</button>
+      <button class="btn-link m-h-3 t-c-6">
+        <i class="fas fa-shopping-cart f-8"></i>
+      </button>
+      <button class="btn-link m-h-3 t-c-6">
```

Comment:
These classes m-h-3 and t-c-6 can be given more readable names for better understanding of the code

Reply:
these are general classes , it will get lengthy if I give it a long name.